src/lib.rs

```diff
@@ -7,6 +7,8 @@ pub mod config;
 mod contracts;
 mod database;
 mod ethereum;
+
+mod identity;
```

Comment:
Please rename `identity.rs` to `identity/mod.rs` we've decided recently to use mod named modules

Reply:
ditto `identity_tree`. as soon as a module has submodules, let's move it into directory for consistency